config.go

```diff
@@ -642,6 +650,7 @@ type ConfigTemplate struct {
 	Backup         bool          `json:"backup" mapstructure:"backup"`
 	LeftDelim      string        `json:"left_delimiter" mapstructure:"left_delimiter"`
 	RightDelim     string        `json:"right_delimiter" mapstructure:"right_delimiter"`
+	Wait           *watch.Wait   `json:"wait" mapstructure:"wait"`
```

Comment:
Is there anything guarding against a negative wait time, or that `max` is less than `min`?  I didn't see anything in this diff that indicated those inputs were tested.